install/gcp/up.sh

```diff
@@ -15,89 +15,76 @@ gcloud services enable artifactregistry.googleapis.com
 
 export CLUSTER_NAME=substratus
 if ! gcloud container clusters describe ${CLUSTER_NAME} --location ${REGION} -q >/dev/null; then
-gcloud container clusters create ${CLUSTER_NAME} --location ${REGION} \
-  --machine-type n2d-standard-8 --num-nodes 1 --min-nodes 1 --max-nodes 5 \
-  --node-locations ${ZONE} --workload-pool ${PROJECT_ID}.svc.id.goog \
-  --enable-image-streaming --enable-shielded-nodes --shielded-secure-boot \
-  --shielded-integrity-monitoring --enable-autoprovisioning \
-  --max-cpu 960 --max-memory 9600 --ephemeral-storage-local-ssd=count=2 \
-  --autoprovisioning-scopes=logging.write,monitoring,devstorage.read_only,compute \
-  --addons GcsFuseCsiDriver
+	gcloud container clusters create ${CLUSTER_NAME} --location ${REGION} \
```

Comment:
this messes up the docs rendering so would prefer original formatting instead of so many spaces in front

Reply:
Editor side-effect, reverted